libraries/AP_Mount/AP_Mount_Alexmos.cpp

```diff
@@ -102,15 +103,31 @@ bool AP_Mount_Alexmos::has_pan_control() const
     return _gimbal_3axis;
 }
 
-// send_mount_status - called to allow mounts to send their status to GCS using the MOUNT_STATUS message
-void AP_Mount_Alexmos::send_mount_status(mavlink_channel_t chan)
+// send a GIMBAL_DEVICE_ATTITUDE_STATUS message to GCS
+void AP_Mount_Alexmos::send_gimbal_device_attitude_status(mavlink_channel_t chan)
 {
     if (!_initialised) {
         return;
     }
 
+    // request attitude from gimbal
     get_angles();
-    mavlink_msg_mount_status_send(chan, 0, 0, _current_angle.y*100, _current_angle.x*100, _current_angle.z*100, _mode);
+
+    // construct quaternion
+    Quaternion q;
+    q.from_euler(radians(_current_angle.x), radians(_current_angle.y), radians(_current_angle.z));
+    const float quat_array[4] = {q.q1, q.q2, q.q3, q.q4};
+
+    mavlink_msg_gimbal_device_attitude_status_send(chan,
+                                                   0,   // target system
+                                                   0,   // target component
+                                                   AP_HAL::millis(),    // autopilot system time
+                                                   get_gimbal_device_flags(),
+                                                   quat_array,      // attitude expressed as quaternion
+                                                   AP::logger().quiet_nanf(),    // roll axis angular velocity (NaN for unknown)
```

Comment:
We use `std::numeric_limits<double>::quiet_NaN()` elsewhere when constructing mavlink packets.  Pretty sure we don't want to be using something from `AP_Logger` when constructing mavlink packets.

Reply:
Txs for the review,fixed!